lib/node_modules/@stdlib/fft/base/fftpack/lib/passfb.js

```diff
@@ -0,0 +1,242 @@
+*       notice, this list of conditions, and the disclaimer below in the
+*       documentation and/or other materials provided with the
+*       distribution.
+*
+* THIS SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+* EXPRESS OR IMPLIED, INCLUDING, BUT NOT LIMITED TO THE WARRANTIES OF
+* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+* NONINFRINGEMENT. IN NO EVENT SHALL THE CONTRIBUTORS OR COPYRIGHT
+* HOLDERS BE LIABLE FOR ANY CLAIM, INDIRECT, INCIDENTAL, SPECIAL,
+* EXEMPLARY, OR CONSEQUENTIAL DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+* ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+* CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS WITH THE
+* SOFTWARE.
+* ```
+*/
+
+'use strict';
+
+// MODULES //
+
+var c1Ref = require( './c1_ref.js' );
+var c2Ref = require( './c2_ref.js' );
+var chRef = require( './ch_ref.js' );
+var ch2Ref = require( './ch2_ref.js' );
+var ccRef = require( './cc_ref.js' );
+
+
+// MAIN //
+
+/**
+* Performs a pass of the FFT algorithm.
+*
+* @private
+* @param {Float64Array} nac - Number of FFT passes.
+* @param {number} ido - Dimension order for input/output arrays.
+* @param {number} ip - Number of sub-steps or prime factors in the FFT.
+* @param {number} l1 - Length parameter related to the FFT stage.
+* @param {number} idl1 - Stride related to the `l1` parameter.
+* @param {Float64Array} cc - Input array containing complex data for FFT computation.
+* @param {Float64Array} c1 - Intermediate array for FFT computations.
+* @param {Float64Array} c2 - Secondary intermediate array for FFT computations.
+* @param {Float64Array} ch - Output array for storing processed FFT data.
+* @param {Float64Array} ch2 - Secondary output array for storing processed FFT data.
+* @param {Float64Array} wa - Twiddle factor array used in FFT calculations.
+* @param {number} fsign - Sign factor indicating the direction of the FFT (e.g., +1 for forward, -1 for inverse).
+* @returns {void}
+*/
+function passfb ( nac, ido, ip, l1, idl1, cc, c1, c2, ch, ch2, wa, fsign ) {
+	var ch2_offset;
+	var ch_offset;
+	var cc_offset;
+	var c1_offset;
+	var c2_offset;
+	var ipp2;
+	var idij;
+	var idlj;
+	var idot;
+	var ipph;
+	var idj;
+	var idl;
+	var inc;
+	var idp;
+	var wai;
+	var war;
+	var jc;
+	var lc;
+	var ik;
+	var i;
+	var j;
+	var k;
+	var l;
+
+	// Parameter adjustments...
+	ch_offset = 1 + ( ido * ( 1 + l1 ) );
+	c1_offset = 1 + ( ido * ( 1 + l1 ) );
+	cc_offset = 1 + ( ido * ( 1 + ip ) );
+	ch2_offset = 1 + idl1;
+	c2_offset = 1 + idl1;
+
+	// Function body:
+	idot = ido / 2;
+	ipp2 = ip + 2;
+	ipph = ( ip + 1 ) / 2;
+	idp = ip * ido;
+	if ( ido >= l1 ) {
+		for ( j = 2; j <= ipph; ++j ) {
+			jc = ipp2 - j;
+			for ( k = 1; k <= l1; ++k ) {
+				for ( i = 1; i <= ido; ++i ) {
+					ch[ chRef( i, k, j, l1, ido ) - ch_offset ] = cc[ ccRef( i, j, k, ip, ido ) - cc_offset ] + cc[ ccRef( i, jc, k, ip, ido ) - cc_offset ]; // eslint-disable-line max-len
```

Comment:
Just disable this lint rule at the top-level, so you are not repeatedly having to disable it.